frontend/src/views/applications/OperatingCostsView.vue

```diff
@@ -124,8 +166,8 @@ export default {
     isRentLease() {
       return this.model.facilityType === FACILITY_TYPES.RENT_LEASE
     },
-    isDocumentUploaded() {
-      return !this.isRentLease || (this.isRentLease && (!isEmpty(this.documentsToUpload) || !isEmpty(this.uploadedDocuments)))
+    areRequiredDocsUploaded() {
```

Comment:
please correct if I'm wrong. I think the user need to upload both Financial Statement and Balance Sheet, but currently, I only need to upload 1 of them to clear the error. 

Also, I believe they will need to upload those documents no matter which facility type they are.

![image](https://github.com/bcgov/ecc-ofm/assets/119005543/5faaaae9-05c2-4e3b-b482-eb715be5a08f)